ui/packages/catalog-ui-search/src/main/webapp/js/extensions/backbone.partialAssociatedModel.js

```diff
@@ -88,8 +88,7 @@ export default Backbone.AssociatedModel.extend({
         this.relations
           .filter(relation => isRelationPartial(this, relation))
           .map(relation => {
-            if (relation.makeSomethingUp) {
-            } else {
+            if (!relation.makeSomethingUp) {
```

Comment:
✏️ I would actually suggest keeping the predicate as it was and instead returning from it. That way you could collapse the nesting, as there's already plenty of it here, making the code a touch easier to read.